mne/preprocessing/xdawn.py

```diff
@@ -7,11 +7,12 @@
 
 import numpy as np
 from scipy import linalg
+from sklearn.preprocessing import LabelEncoder
```

Comment:
I wrote the `import` statement below in the function scope and tests still failed locally.